etcdstore.go

```diff
@@ -0,0 +1,185 @@
+package etcdstore
+
+import (
+	"context"
+	"encoding/base32"
+	"errors"
+	"fmt"
+	"go.etcd.io/etcd/clientv3"
+	"net/http"
+	"strings"
+
+	"github.com/gorilla/securecookie"
+	"github.com/gorilla/sessions"
+)
+
+var (
+	ErrNilClient   = errors.New("etcd client is nil")
```

Comment:
May add some explanations for these two errors.

Reply:
These errors are not used. So I delete them.